examples/http-error-handling/http_error_handling.bal

```diff
@@ -31,15 +31,18 @@ service class ResponseErrorInterceptor {
 // Creates a new `ResponseErrorInterceptor`.
 ResponseErrorInterceptor responseErrorInterceptor = new;
 
+// Creates the interceptors pipeline.
+http:CreateInterceptorsFunction interceptorsFunction = isolated function () returns ResponseErrorInterceptor {
```

Comment:
This one has more information like we are creating a type of `http:CreateInterceptorsFunction`. But we can also do this right?
```bal
isolated function createInterceptors() returns ResponseErrorInterceptor {
     return new ResponseErrorInterceptor();
}

listener http:Listener interceptorListener = new (9090, interceptors = createInterceptors);
```

Reply:
We can do that. however my thought was to show the user what kind of a type does this require so that user can go to the function definition easily.
Im +1 for both options

Reply:
I am ok with the current one, just want to verify this 👍🏼